src/components/NcListItem/NcListItem.vue

```diff
@@ -215,7 +215,7 @@
 						:class="{ 'list-item-content__main--oneline': oneLine }">
 
 						<!-- First line, title and details -->
-						<div class="line-one">
+						<div v-if="!$slots.indicator" class="line-one">
```

Comment:
I am a bit puzzled by this. If there is an indicator slot, the title will be shown nowhere?

Reply:
right